### PR TITLE
RIA-3163 payment after submit appeal

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3163](https://tools.hmcts.net/jira/browse/RIA-3163)

### Change description ###
Make a payment after submitting the appeal for EA/HU/PA appeal types

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Lee365 — APPROVED** [review]

File: e2e/features/submit-appeal-and-make-payment.feature [hunk 1]
```diff
@@ -0,0 +1,308 @@
+Feature: Leadership judge record the appellant decision and reasons
+
```
Comment (Srihari-Nakkala) [inline e2e/features/submit-appeal-and-make-payment.feature:2]: @bradsorour i think we need to change the Title of Feature

Comment (bradsorour) [reply e2e/features/submit-appeal-and-make-payment.feature:2]: Yep, that doesn't make sense.

Comment (bradsorour) [reply e2e/features/submit-appeal-and-make-payment.feature:2]: Fixed.

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**